aten/src/ATen/native/native_functions.yaml

```diff
@@ -10541,6 +10541,21 @@
   python_module: nn
   variants: function
 
+- func: pad(Tensor self, Tensor pad_width, str mode='constant', *, Tensor? constant_values=None) -> Tensor
+  variants: function, method
```

Comment:
Out of an abundance of caution let's not add a method variant of this for now to avoid confusion with torch.nn.functional.pad.

Reply:
cc @jbschlosser